decryptDefaultKey/decryptDefaultKey.ino

```diff
@@ -0,0 +1,82 @@
+#include <AESLib.h>
+
+#define BAUD 9600
+
+AESLib aesLib;
+String kim="r9Huf1ov92qFJMtG";
+// AES Encryption Key
+byte aesKey[] = {kim[0], kim[1], kim[2], kim[3], kim[4], kim[5], kim[6], kim[7], kim[8], kim[9], kim[10], kim[11], kim[12], kim[13], kim[14], kim[15]};
+// General initialization vector (use your own)
+byte aesIv[16] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
Please dont use all zeros, AA, BB etc may be better

Reply:
ok